Sources/Sentry/include/SentryBinaryImageCache.h

```diff
@@ -0,0 +1,23 @@
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface BinaryImageInfo : NSObject
```

Comment:
`h`: To avoid naming collisions.
```suggestion
@interface SentryBinaryImageInfo : NSObject
```

Reply:
Good point, I create this as I static struct at first.